src/foam/layout/Section.js

```diff
@@ -12,14 +12,27 @@ foam.CLASS({
     Provides model data sectioned viewing when using section views.
     Used for sectioning/grouping model properties and actions.
 
+    Section title and subtitle functions have a helper method (1st arg)
+    to evaluate messages or values that are template literals.
+
+    Template literal messages should be considered over constructing strings
+    within these functions. Locale translation support will have an easier time
+    understanding and applying the appropriate translations.
+
+    These functions are also executed when model data changes,
+    supporting dynamic section labeling and data awareness.
+
     Example:
     foam.CLASS({
       name: 'myModel',
+      messages: [
+        { name: 'myMessage', message: 'This message will evaluate a template literal \${myProp}' }
+      ],
       sections: [
         {
           name: 'mainSection'
-          title: function(myProp) {
-            return myProp ? 'Set this title if myProp true' : 'Set this title if myProp false';
+          title: function(eval, data) {
```

Comment:
Eval is a poor name choice, because people and audit-tools will think we're calling the JS-eval method. How about 'expand' or something else?

Reply:
Ahh - yes very true, will update to evaluateMessage